src/PrimaryGeneration.cpp

```diff
@@ -47,25 +50,16 @@
 
 PrimaryGeneration::PrimaryGeneration()
   : G4VUserPrimaryGeneratorAction(),
-    decay_at_time_zero_(false),
-    isotropic_(false),
-    override_vertex_position_(false),
-    vertex_x_(2.3/2),
-    vertex_y_(6.0/2),
-    vertex_z_(3.6/2),
-    particle_gun_(0)
+    particle_gun_(0)//,
+    //isotropic_(false),
+    //decay_at_time_zero_(false),
+    //override_vertex_position_(false),
+    //vertex_x_(0),
+    //vertex_y_(0),
+    //vertex_z_(0),
+    //printParticleInfo_(false),
+    //particleType_("")
```

Comment:
Don't include commented out code in the repository.

Reply:
fixing in bugfix/v3.0.0_revisions branch